doc/changes/latest.inc

```diff
@@ -37,6 +37,7 @@ Enhancements
 
 Bugs
 ~~~~
+- Fix bug where :func:`mne.io.read_raw_cnt` imports unnecessary durations (:gh:`11828` by `Jacob Woessner`)
```

Comment:
are you a new contributor @withmywoessner ? if not can you put your name below the new contrib or if so can you put the newcontrib tag as the line below?

Reply:
I have already contributed before.

Reply:
@withmywoessner don't forget the first part of the conditional above:

> if not can you put your name below the new contrib

We try to have the `:newcontrib:` at the top of each section, so this line needs to drop down a few lines